jsonrpc/src/errors/errors.json

```diff
@@ -0,0 +1,13 @@
+{
+  "ContentNotFoundError": {
+    "code": -32099,
```

Comment:
We are using `39001` for this in the code (see PR https://github.com/ethereum/portal-network-specs/pull/276/files)

Reply:
Have updated

Reply:
I'm still seeing the wrong error code here? Maybe the change didn't get merged? And if we're going with `39001` for `ContentNotFoundError` then we should go with something like `39002` for `ContentNotFoundErrorWithTrace`?

Reply:
Ugh, sorry, the commit didn't grab the changes in this file.  Should be correct now.